frontend/src/components/message/Message.tsx

```diff
@@ -38,7 +38,9 @@ const Message: FC<{ message: IMessage }> = ({ message }) => {
 			ref={outsideRef}
 			onMouseOver={handleMouseOver}
 			onMouseOut={handleMouseOut}>
-			<MessageAuthorImage src={user} />
+			<MessageAuthorImage
```

Comment:
prettier reformat it like that, u have good config?

Reply:
there, fixed